CompuCell3D/core/pyinterface/FiPyInterface/FiPyInterface.cpp

```diff
@@ -1,7 +1,7 @@
 #include <iostream>
```

Comment:
I would not mind getting rid of this module (FiPy) . It was experimental a while ago and I don't think anybody uses it